bigbluebutton-html5/imports/ui/components/whiteboard/slide-controls/component.jsx

```diff
@@ -0,0 +1,296 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './styles.scss';
+import Button from '/imports/ui/components/button/component';
+import { callServer } from '/imports/ui/services/api/index.js';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+
+export default class SlideControls extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { sliderValue: 100 };
+
+    this.nextSlide = this.nextSlide.bind(this);
+    this.previousSlide = this.previousSlide.bind(this);
+    this.skipToSlide = this.skipToSlide.bind(this);
+    this.clickHandler = this.clickHandler.bind(this);
+    this.handleValuesChange = this.handleValuesChange.bind(this);
+  }
+
+  handleValuesChange(event) {
+    this.setState({ sliderValue: event.target.value });
+  }
+
+  // Change to the next slide
+  nextSlide() {
+    const currentSlideNum = this.props.currentSlideNum;
+    if (this.nextSlideAvailable()) {
+      callServer('publishSwitchToXSlideMessage', currentSlideNum + 1);
+    }
+  }
+
+  nextSlideAvailable() {
+    const currentSlideNum = this.props.currentSlideNum;
+    let nextSlideAvailable = true;
+
+    if (currentSlideNum == this.props.numberOfSlides) {
+      nextSlideAvailable = false;
+    }
+
+    return nextSlideAvailable;
+  }
+
+  // Change to the previous slide
+  previousSlide() {
+    const currentSlideNum = this.props.currentSlideNum;
+
+    if (this.prevSlideAvailable()) {
+      callServer('publishSwitchToXSlideMessage', currentSlideNum - 1);
+    }
+  }
+
+  prevSlideAvailable() {
+    const currentSlideNum = this.props.currentSlideNum;
+    let prevSlideAvailable = true;
+
+    if (currentSlideNum == 1) {
+      prevSlideAvailable = false;
+    }
+
+    return prevSlideAvailable;
+  }
```

Comment:
Your prevSlideAvailable function could be reduced down to one line "return this.props.currentSlideNum > 1". No variable creation needed.